source/languages/en/riak/community/index.md

```diff
@@ -11,116 +11,79 @@ moved: {
-Whether you need help or simply want to get involved, we are easy to
-contact.
+There is a vibrant, passionate and growing community forming around Riak. Whether you need help or simply want to get involved, we are easy to contact.
 
-## Help
+## Platforms
+We're big on answering questions for beginner and veteran users alike. To make sure we can answer them, we curate a number of discussions on a daily basis. 
 
 ### Mailing List
 
 The [Riak Users Mailing
 List](http://lists.basho.com/mailman/listinfo/riak-users_lists.basho.com)
-is highly trafficked and a great resource for technical discussions,
-Riak issues and questions, and community events and announcements.
+is highly trafficked and a great resource for technical discussions, Riak issues and questions, and community events and announcements.
 
-### Code
+We pride ourselves on answering every email that comes over the Riak User mailing list. [Sign up](http://lists.basho.com/mailman/listinfo/riak-users_lists.basho.com) and send away. If you prefer points for your questions, you can always [tag Riak on StackOverflow](http://stackoverflow.com/questions/tagged/riak). 
 
-Basho uses GitHub for the development of [all of our
-code](http://github.com/basho/) and we also make use of GitHub for the
-submission and integration of all internal code and community pull
-requests.
 
-Basho also maintains a frequently updated page of [[community-developed
-projects|Community Projects]], including GUIs, backup tools, and more.
+### Code
 
-### Riak on IRC
+Basho uses GitHub for the development of all of our
+code. Our core products are developed [under the Basho organization](http://github.com/basho/). We also have [Basho Labs](http://github.com/basho-labs/), where we curate the rich ecosystem of contributors from our community. We also make use of GitHub for the submission and integration of all internal code and community pull requests. If you have general feedback beyond the scope of a single repo, open an issue under [The Riak Community](https://github.com/basho-labs/the-riak-community/issues).
 
-The **#riak** IRC room on irc.freenode.net is a great place for realtime
-help with your Riak issues and questions.
+### Events on Meetup.com
 
-### The Basho Blog
+We have local meetups everywhere from Paris to San Francisco and back again. To find the one closest to you, search for [Riak on Meetup.com](http://riak.meetup.com/).
 
-Members of the Basho Team blog frequently on [[The Basho
-Blog|http://blog.basho.com]].
+### Riak on IRC
 
-### The Riak Recap
+The [#riak IRC room on irc.freenode.net](http://irc.lc/freenode/riak) is a great place for realtime help with your Riak issues and questions.
 
-The Riak Recap is a periodic email to the Riak Users Mailing List to
-keep list subscribers up-to-date on what’s happening in and around the
-Riak Community.
+### The Basho Blog
 
-### Riak Function Contrib
+Members of the Basho Team blog frequently on [The Basho
```

Comment:
Maybe "post frequently on..." I had to read that sentence several times before I figured out that blog was both a noun and verb.